multistats.py

```diff
@@ -16,6 +16,17 @@
 api= wandb.Api(timeout=60)
 wandb.login(anonymous="allow")
 
+import json
+# Store the original json.loads
+original_json_loads = json.loads
+
+def relaxed_json_loads(s, *args, **kwargs):
+    kwargs.pop("strict", None)  # remove the strict argument if it exists
+    return original_json_loads(s, strict=False, *args, **kwargs)
```

Comment:
why not call it explicitly like this in every `json.loads call` ?? or use this `relaxed_json_loads`.
Even though the approach is very nice in shape. I think it obscures what's really happening, and that's important for maintenance and readability of the code.

From my point of view, I'd say, let's make the call explicit as it is in this line, or use the function `relaxed_json_loads`, so we know something different is happening

What do you think about this?

Reply:
Yea it's not my preferred solution at all, however the issue actual originates inside the wandb codebase - they enforce strict mode when pulling assets from their databases. We never actually call json decide ourselves..
I propose we put it either here or shamefully in a Util file for now

Reply:
So it's a way of overriding json globally